finder-app/finder.sh

```diff
@@ -37,7 +37,7 @@ do
 	then
 	
 		# Check if the file contains the search string
-		if grep -q "$searchstr" "$i"; then
+		if sudo grep -q "$searchstr" "$i"; then
```

Comment:
@BhaktiRamani sudo shouldn't be necessary here, there's a chance it may cause tests to fail in assignment 3.